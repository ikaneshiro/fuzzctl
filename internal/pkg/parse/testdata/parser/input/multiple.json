{
  "version": "0.1",
  "workflow": {
    "name": "MultipleJobs",
    "jobs": {
      "date": {
        "image": "library://alpine:latest",
        "command": [
          "date"
        ]
      },
      "hello": {
        "image": "library://alpine:latest",
        "command": [
          "echo",
          "-n",
          "Hello, world!"
        ]
      }
    }
  }
}