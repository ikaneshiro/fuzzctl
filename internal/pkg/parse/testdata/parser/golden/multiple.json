{
	"name": "MultipleJobs",
	"jobs": [
		{
			"name": "date",
			"image": "library://alpine:latest",
			"command": [
				"date"
			],
			"requires": null
		},
		{
			"name": "hello",
			"image": "library://alpine:latest",
			"command": [
				"echo",
				"-n",
				"Hello, world!"
			],
			"requires": null
		}
	]
}